js/pages/concept-sets/conceptset-manager.js

```diff
@@ -130,7 +131,10 @@ define([
 				{
 						title: 'Included Concepts',
 						componentName: 'included-conceptsets',
-						componentParams: params,
+						componentParams: {
```

Comment:
Why do we need to pass this observable via props? Why not to define dedicated dispose in `included-conceptsets.js`?